content/1_docs/2_cookbook/9_setup/0_php-debugging-with-xdebug/recipe.txt

```diff
@@ -189,6 +189,8 @@ In this first step, the `$notes` and `$tag` variable are still uninitialized.
 
 If we now click on `Step over` in the debug toolbar (or press <kbd>F10</kbd>), we can see how the content in the debug panel changes. Now the `$tag` variable is no longer uninitialized, but changes to an empty string.
 
+(screenshot: tag-variable.png)
```

Comment:
Shouldn't the screenshot be below the description?

Reply:
🙈 😢

Reply:
If it's still messed up now, I'll just go to bed.